base/linalg/schur.jl

```diff
@@ -111,8 +111,7 @@ schur(A::Symmetric) = schur(full(A))
 schur(A::Hermitian) = schur(full(A))
 schur(A::UpperTriangular) = schur(full(A))
 schur(A::LowerTriangular) = schur(full(A))
-
-
+schur(A::Tridiagonal) = schur(full(A))
```

Comment:
Of course. I just added these definitions to make it work for now. I plan to work on `schur()` after this PR is closed.